gui/guihtml/src/TGHtmlUri.cxx

```diff
@@ -227,7 +227,7 @@ char *TGHtmlUri::BuildUri()
    if (fZAuthority && fZAuthority[strlen(fZAuthority)-1] != '/' &&
       !(fZPath && fZPath[0] == '/')) {
       // coverity[secure_coding]
-      strcat(z, "/");
+      strlcat(z, "/", sizeof(z));
```

Comment:
Here is wrong.
In begin of function you need to remember allocated size and use it here.

Reply:
As previous comments, I will revert and add `// NOLINT`